src/index.js

```diff
@@ -29,16 +28,10 @@ class FeathersError extends Error {
       message = msg;
     }
 
-    if (data) {
-      // NOTE(EK): To make sure that we are not messing
-      // with immutable data, just make a copy.
-      // https://github.com/feathersjs/feathers-errors/issues/19
-      newData = Object.assign({}, data);
-
-      if (newData.errors) {
-        errors = newData.errors;
-        delete newData.errors;
-      }
+    if (data && data.errors) {
```

Comment:
There is a side effect here. If `data.errors` is `undefined` then the data object won't be cloned. If that data object is modified elsewhere it also be modified here.

The intention was that once the `data` object is passed in when creating a new Error that data object should be immutable.